PokemonGo/RocketAPI/Window/PokeUi.cs

```diff
@@ -96,8 +97,8 @@ private void PokeUi_Load(object sender, EventArgs e)
 
 
                     this.listView1.Items.Add(listViewItem);
-
-
+		    this.Text = "PokeUi " + countpkmn + "/" + profile.Profile.PokeStorage;
```

Comment:
Moved out. It was into the for each just to make the incrementation effect in the title.